lib/foreman_hooks/util.rb

```diff
@@ -8,14 +8,26 @@ def render_hook_type
       when Host::Managed
         'host'
       else
-        self.class.name.downcase
+        self.class.name.demodulize.downcase
```

Comment:
yes, the render_hook_type is used to look up the v2 api which is called. If you use the hook 

config/hooks/katello/product/before_create/01_integrate_catalog.sh

as an example, the view is located at 

katello/api/v2/products/show not in /katello/api/v2/katello/products/show which this would result in without the demodulize. I tested with the above and saw that the event, the id, and the full json was passed into the hook.